guides/common/modules/proc_using-a-third-party-repository.adoc

```diff
@@ -0,0 +1,19 @@
+[id="Using_a_Third_Party_Repository_{context}"]
+= Using a Third Party Repository
+
+In this scenario, you use a third party repository to sync content.
+
+.On a connected server
+. In the connected server of the {ProjectWebUI}, navigate to *Content* > *Product*.
+. Click on the third party product.
+. Click over the third party repository.
+On this page, you can see the link in front of *Published At:*.
+Copy the information in the link.
```

Comment:
Is this about copying the link itself or copying the stuff it takes you to when clicking the link?